src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  
```

Comment:
this file can be reverted. No need in changing it in this PR.

Reply:
Done